CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
-## [unreleased]
+## [4.61]
```

Comment:
Did you start this PR from the new minor release branch?

Reply:
Ooh, quite possible! Will check!